packages/python/plotly/_plotly_utils/basevalidators.py

```diff
@@ -2647,10 +2646,6 @@ def get_trace_class(self, trace_name):
     def validate_coerce(self, v, skip_invalid=False, _validate=True):
         from plotly.basedatatypes import BaseTraceType
 
```

Comment:
```suggestion

        # Import Histogram2dcontour, this is the deprecated name of the
        # Histogram2dContour trace.
        from plotly.graph_objs import Histogram2dcontour
```
Looks like along with my previous comment we need to put this back in 😅